packages/react-devtools-shared/src/utils.js

```diff
@@ -996,11 +996,6 @@ export function backendToFrontendSerializedElementMapper(
   };
 }
 
-// Chrome normalizes urls like webpack-internals:// but new URL don't, so cannot use new URL here.
-export function normalizeUrl(url: string): string {
-  return url.replace('/./', '/');
-}
-
```

Comment:
Platform specific code so we inlined this only to the specific platforms that need normalization.